debian/xenial/kubelet/debian/control

```diff
@@ -1,16 +1,16 @@
 Source: kubelet
 Section: misc
 Priority: optional
-Maintainer: Mike Danese <mikedanese@google.com>
+Maintainer: Kubernetes Authors <kubernetes-dev@google.com>
 Build-Depends: debhelper (>= 8.0.0)
 Standards-Version: 3.9.4
 Homepage: https://kubernetes.io
 Vcs-Git: https://github.com/kubernetes/kubernetes.git
 Vcs-Browser: https://github.com/kubernetes/kubernetes
 
 Package: kubelet
-Architecture: amd64
-Depends: docker-engine (= 1.11.2-0~xenial), iptables (>= 1.4.21), k8s-cni (= 0.0.1-0),
- ${shlibs:Depends}, ${misc:Depends}
+Architecture: {{ .Arch }}
+Depends: docker-engine (>= 1.11.2-0~xenial), iptables (>= 1.4.21), kubernetes-cni (>= 0.0.1-0),
```

Comment:
Can we remove this dep for the other arches as the package is often called something else...

e.g. the `docker-hypriot` package exists for earlier docker versions (<=1.11), which then finally became replaced with the official armhf `docker-engine` v1.12.1

Otherwise installing this deb on another arch will fail (especially with the `~xenial`  at the end)